README.md

```diff
@@ -58,148 +65,67 @@ spec:
 EOF
 ```
 
+Once the catalog is unpacked successfully, its content will be available for installation.
+
 ```bash
 # Wait for the ClusterCatalog to be unpacked
 kubectl wait --for=condition=Unpacked=True clustercatalog/operatorhubio --timeout=60s
 ```
 
-### Install Cluster Extension
+### Install a Cluster Extension
+
+For simplicity, the following example manifest includes all necessary resources to install the ArgoCD operator. 
+The manifest includes installation namespace, installer service account and associated minimal set of RBAC permissions 
+needed for installation, and the ClusterExtension resource, which specifies the name and version of the extension to install.
+More information on installing extensions can be found [here](./docs/Tasks/installing-an-extension.md).
 
 ```bash
 # Apply the sample ClusterExtension. Manifest already includes
 # namespace and adequately privileged service account
 kubectl apply -f config/samples/olm_v1alpha1_clusterextension.yaml
```

Comment:
should we make this a direct link so that users can use this file without having to clone the operator-controller repo first?

Reply:
I'm an idiot - I focused on the green part rather than the line. My bad.